plugin/trino-pinot/pom.xml

```diff
@@ -14,15 +14,25 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.pinot.version>0.12.1</dep.pinot.version>
+        <dep.pinot.version>1.1.0</dep.pinot.version>
+        <air.test.jvm.additional-arguments>${air.test.jvm.additional-arguments.default}
+            --add-exports=java.base/jdk.internal.ref=ALL-UNNAMED
+            --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
+            --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
+            --add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED
+            --add-opens=jdk.compiler/com.sun.tools.javac=ALL-UNNAMED
+            --add-opens=java.base/java.lang=ALL-UNNAMED
+            --add-opens=java.base/java.lang.reflect=ALL-UNNAMED
+            --add-opens=java.base/java.io=ALL-UNNAMED
+            --add-opens=java.base/java.util=ALL-UNNAMED</air.test.jvm.additional-arguments>
```

Comment:
> @ebyhr Are these arguments required not only in tests but also in actual Trino clusters?
> @elonazoulay Yes

How did you confirm these JVM options are required in actual clusters? I ran `TestPinotConnectorSmokeTest` without additional JVM options and only testTimeBoundary is affected. The failure place is testing helper method (`createAndPopulateMixedCaseHybridTablesAndTopic`), not product path.

Reply:
The failure is on the pinot-path:
```
	at net.openhft.chronicle.core.internal.ClassUtil.getSetAccessible0Method(ClassUtil.java:32)
	at net.openhft.chronicle.core.internal.ClassUtil.<clinit>(ClassUtil.java:13)
	at net.openhft.chronicle.core.Jvm.getField(Jvm.java:541)
	at net.openhft.chronicle.core.Jvm.maxDirectMemory0(Jvm.java:906)
	at net.openhft.chronicle.core.Jvm.<clinit>(Jvm.java:145)
	at org.apache.pinot.segment.spi.utils.JavaVersion.<clinit>(JavaVersion.java:32)
	at org.apache.pinot.segment.spi.memory.PinotDataBuffer.createDefaultFactory(PinotDataBuffer.java:169)
	at org.apache.pinot.segment.spi.memory.PinotDataBuffer.createDefaultFactory(PinotDataBuffer.java:162)
	at org.apache.pinot.segment.spi.memory.PinotDataBuffer.<clinit>(PinotDataBuffer.java:132)
	at org.apache.pinot.segment.local.segment.creator.impl.SegmentDictionaryCreator.writeBytesValueDictionary(SegmentDictionaryCreator.java:268)
	at org.apache.pinot.segment.local.segment.creator.impl.SegmentDictionaryCreator.build(SegmentDictionaryCreator.java:218)
	at org.apache.pinot.segment.local.segment.creator.impl.SegmentColumnarIndexCreator.init(SegmentColumnarIndexCreator.java:184)
	at org.apache.pinot.segment.local.segment.creator.impl.SegmentIndexCreationDriverImpl.build(SegmentIndexCreationDriverImpl.java:228)
	at io.trino.plugin.pinot.BasePinotConnectorSmokeTest.createSegment(BasePinotConnectorSmokeTest.java:669)
	at io.trino.plugin.pinot.BasePinotConnectorSmokeTest.createAndPopulateMixedCaseHybridTablesAndTopic(BasePinotConnectorSmokeTest.java:469)
	at io.trino.plugin.pinot.BasePinotConnectorSmokeTest.createQueryRunner(BasePinotConnectorSmokeTest.java:156)
	at io.trino.testing.AbstractTestQueryFramework.init(AbstractTestQueryFramework.java:113)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:507)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1491)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:2073)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2035)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
```

Seems like a requirement for jvm flags comes from chronicle-hft.

Reply:
I think that it is required but just one flag is needed. I'll push in a second

Reply:
`PinotDataBuffer` is used on the product path

Reply:
PTAL now.

Reply:
Only `--add-opens=java.base/java.lang.reflect=ALL-UNNAMED` is needed